data/repositories/fedora-43.json

```diff
@@ -0,0 +1,34 @@
+{
+  "x86_64": [
+    {
+      "name": "fedora",
+      "metalink": "https://mirrors.fedoraproject.org/metalink?repo=fedora-rawhide&arch=x86_64",
```

Comment:
Maybe worth mentioning in the commit message that we are adding fedora-43 which is currently rawhide? As it is not really labeld fedora-43 in the metalink yet?

Reply:
It says the following, do you mean you want it in the subject line? :)

>  data: fedora 43 repositories
>
> These currently refer to rawhide but naming them rawhide is hard in our
stack.

Reply:
Oh, sorry, I did not see it in the PR description, I have not looked into the individual commit(s). Fine then.